release/readme.html

```diff
@@ -138,13 +140,14 @@ <h5 class="card-title">Yes... and no.</h5>
             <a class="btn btn-primary margin-bottom-6" data-toggle="collapse" href="#collapseIDDIDF" role="button"
                aria-expanded="false"
                aria-controls="collapseIDDIDF">
-                Are 9.5 and 9.4 IDD and IDF files the same?
+                Are 9.6 and 9.5 IDD and IDF files the same?
```

Comment:
Usual updates for new version in the README.  If I make another commit I'll also add an entry for the new deprecation document.